File: SEP3_Client/Mediator/Information/InformationPackage.cs
namespace SEP3_Client.Mediator.Information
{
    public abstract class InformationPackage
    {
        public InformationType informationType { get; set; }
        public string Keyword { get; set; }

        public InformationPackage(InformationType informationType,string keyword)
        {
            this.informationType = informationType;
            Keyword = keyword;
        }

        public InformationType GetInformationType()
        {
            return informationType;
        }

        public string GetKeyword() {
            return Keyword;
        }

        public override string ToString()
        {
            return "[" + informationType + "]";
        }
    }
}

File: SEP3_UserSystem/src/main/java/mediator/information/InformationPackage.java
package mediator.information;

public abstract class InformationPackage {
    private InformationType informationType;
    private String Keyword;

    public InformationPackage(InformationType informationType,String keyword)
    {
        this.informationType = informationType;
        Keyword = keyword;
    }

    public InformationType getInformationType()
    {
        return informationType;
    }

    public String getKeyword() {
        return Keyword;
    }

    @Override public String toString()
    {
        return "[" + informationType + "]";
    }
}
